### DISCUSSION TITLE
error Command failed with exit code 3221225477

### DISCUSSION THREAD

**Discussion by fathi91** [nuxt/nuxt#16793] [Questions]
Good day
after creating a new fresh nuxt project 
i started to get this error  randomly every min or so
"erroe Command failed with exit code 3221225477"
i don't know if this issue related to nuxt and hot reload or not

**Comment by sevillaarvin** [comment]
Seems to be a windows related issue. Try updating your node version.

https://stackoverflow.com/a/58959576
